packages/widget/src/context/config.ts

```diff
@@ -24,6 +27,15 @@ export class ConfigContextProvider extends BaseComponent {
     initialValue: {}
   });
 
+  @property({ attribute: false, type: Object })
+  whitelistedSourceNetworks?: string[];
+
+  @property({ attribute: false, type: Object })
+  whitelistedDestinationNetworks?: string[];
+
+  @property({ attribute: false, type: Object })
+  whitelistedSourceResources?: string[];
```

Comment:
Is there a reason for putting this into context? We are passing it only to the first child (`<sygma-fungible-transfer\`), while context is used to avoid passing it trough multiple levels